apps/dashr-multipage-report/app.R

```diff
@@ -0,0 +1,1916 @@
+                      'horizontal-align' = "middle", 'padding-left' = "40px", 'padding-top' = "20px", 
+                      'margin-left' = "15px", 'margin-top' = "30px")),
+      
+      
+      
+      htmlDiv(list(
+        htmlDiv(list(
+          htmlH6('Viverra, imperdiet, praesent pellentesque', style = list('color' = "#003399")),
+          htmlP(LipP(1, F))
+        ), style = list('border-left' = "5px", 'border-left-style' = "solid", 
+                        "border-left-color" = color_1,
+                        "border-left-width" = "7px", 'padding-left' = "20px", 'margin-top' = "30px")),
+        htmlDiv(list(
+          htmlH6('Facilisis mauris parturient, eget vitae', style = list('color' = "#003399")),
+          htmlP(LipP(1, F))
+        ), style = list('border-left' = "5px", 'border-left-style' = "solid",
+                        "border-left-color" = color_2,
+                        "border-left-width" = "7px", 'padding-left' = "20px", 'margin-top' = "20px")),
+        htmlDiv(list(
+          htmlH6('A suspendisse mauris aliquam tincidunt hac', style = list('color' = "#003399")),
+          htmlP(LipP(1, F))
+        ), style = list('border-left' = "5px", 'border-left-style' = "solid",
+                        "border-left-color" = color_1,
+                        "border-left-width" = "7px", 'padding-left' = "20px", 'margin-top' = "20px")),
+        htmlDiv(list(
+          htmlH6('A elementum lorem dolor aliquam nisi diam', style = list('color' = "#003399")),
+          htmlP(LipP(1, F))
+        ), style = list('border-left' = "5px", 'border-left-style' = "solid",
+                        "border-left-color" = color_2,
+                        "border-left-width" = "7px", 'padding-left' = "20px", 'margin-top' = "20px"))
+        
+      ), style = list('display' = "block", 'float' = "right", 'width' = "400px", 'margin-left' = "10px",
+                      'margin-right' = "15px"))
+      
+    ), className = "subpage")
+  ), className = "page"),
+  
+  ## Page 2
+  htmlDiv(list(
+    
+    htmlDiv(list(
+      
+      # Heading ... creat function
+      htmlDiv(list(
+        htmlH1('LOREM IPSUM')
+      ), style = list('background-color' = color_1, 'color' = "white", 'width' = "200px",
+                      'height' = "160px", 'padding' = "20px", 'margin-left' = "50px",
+                      'margin-top' = "-38px")),  #'float' = "left"
+      
+      htmlDiv(list(
+        htmlP(LipP(2, F), style = list('margin-top' = "40px")),
+        htmlP(LipP(1, F), style = list('margin-top' = "20px")),
+        htmlP(LipP(1, F), style = list('margin-top' = "20px"))
+      ), style = list('margin-left' = "80px"), className = list("five columns")), 
+      
+      htmlDiv(list(
+        htmlP(LipP(1, F), style = list('margin-top' = "40px")),
+        htmlP(LipP(2, F), style = list('margin-top' = "20px"))
+      ), style = list('left-margin' = "30px"), className = list("five columns"))
```

Comment:
Passing a list to `className` leads to this error:
```
Invalid argument `className` passed into Div. Expected `string`. Was supplied type `array`. Value provided: [ "five columns" ]
```

replace with `className = "five columns"`